support-frontend/app/controllers/Application.scala

```diff
@@ -533,6 +534,7 @@ class Application(
         productCatalog = productCatalog,
         allProductPrices = allProductPrices,
         user = request.user,
+        homeDeliveryPostcodes = Some(PaperValidation.M25_POSTCODE_PREFIXES),
```

Comment:
This was missing from previous PRs so the logic to work out whether a postcode was inside the M25 wasn't working properly.